sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -59,17 +60,25 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(options, nameof(options));
 
+            string scope = options.Audience + "/.default";
+
+            //if no Audience is provided, default to Azure Public Cloud
+            if (options.Audience == null)
+            {
+                scope = TextAnalyticsAudience.AzureResourceManagerPublicCloud + "/.default";
+            }
+
             _baseUri = endpoint;
             _clientDiagnostics = new TextAnalyticsClientDiagnostics(options);
             _options = options;
 
-            var pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, Constants.DefaultCognitiveScope));
+            var pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, scope));
             _serviceRestClient = new TextAnalyticsRestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri, TextAnalyticsClientOptions.GetVersionString(options.Version));
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AzureKeyCredential"/>
-        /// class for the specified service instance.
+        /// class for the specified service instance in Azure Public Cloud.
```

Comment:
I don't think that we need to make this distinction here. Given that the majority of use is in the public cloud, my concern is that this may cause more confusion to the 90% case than it would be helpful for the 10% case. My vote would be to just demonstrate use in other clouds via samples.